string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,5 @@
+/* exported isVowel */
+function isVowel(char) {
+  const regex = /[aeiou]/i;
+  return regex.test(char);
+}
```

Comment:
Do you know how `regex` actually works? If you do, then great, but otherwise I encourage you not to google the answers.

Here's how I would've solved this in a non regex way
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```

Reply:
Yep I did about half of the regex exercises on FCC so I have some idea, there was one exercise that was specifically about vowels so it triggered my memory and I had to look up the exercise again. Thanks for checking!

Reply:
No problem! Just wanted to make sure! I had someone in my last cohort straight up copy this answer from Google, so just checking 😂